auditlog/tests/test_auditlog.py

```diff
@@ -368,3 +368,173 @@ def setUp(self):
     def tearDown(self):
         self.groups_rule.unlink()
         super(TestAuditlogFullCaptureRecord, self).tearDown()
+
+
+class AuditLogRuleTestCommon(TransactionCase):
```

Comment:
please use SavepointCase and move setUp code to setUpClass + set tracking_disable on the whole env

Reply:
Corrections done in test file